modules/t/LDFeatureContainerAdaptor.t

```diff
@@ -129,21 +129,87 @@ foreach my $ld_value (@$ld_values) {
   my $variation_name2 = $ld_value->{variation_name2};
   my $r2 = $ld_value->{r2};
   my $d_prime = $ld_value->{d_prime};
-  cmp_ok($result->{$variation_name2}->{r2}, '==', $r2, " fetch_by_VariationFeature r2 for $variation_name2 and rs1333049");
-  cmp_ok($result->{$variation_name2}->{d_prime}, '==', $d_prime, " fetch_by_VariationFeature d_prime for $variation_name2 and rs1333049");
+  cmp_ok($result->{$variation_name2}->{r2}, '==', $r2, "fetch_by_VariationFeature r2 for $variation_name2 and rs1333049");
+  cmp_ok($result->{$variation_name2}->{d_prime}, '==', $d_prime, "fetch_by_VariationFeature d_prime for $variation_name2 and rs1333049");
 }
 
-#fetch_by_VariationFeatures
-#{ population_id => 102179, vf1_name => 'rs1333047', vf2_name => 'rs72655407', r2 => 0.063754, d_prime => 0.999996 },
+#fetch_by_VariationFeature, population is not defined
```

Comment:
Good job adding all these extra tests - this module was hideously undertested before.